src/components/Gradebook/StatusAlerts.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import { StatusAlert } from '@edx/paragon';
+
+import { closeBanner } from '../../data/actions/grades';
+
+export class StatusAlerts extends React.Component {
+  get courseGradeFilterAlertDialogText() {
+    let dialogText = '';
+    if (!this.props.isMinCourseGradeFilterValid) {
+      dialogText += 'Minimum course grade value must be between 0 and 100. ';
+    }
+    if (!this.props.isMaxCourseGradeFilterValid) {
+      dialogText += 'Maximum course grade value must be between 0 and 100. ';
+    }
+    return dialogText;
+  }
+
+  render() {
+    return (
+      <div>
```

Comment:
Nit: this can probably be a react fragment `<>` instead of a div to avoid any changes to the output HTML.